apps/deepl_integration/handlers.py

```diff
@@ -188,13 +188,14 @@ def send_trigger_request_to_extractor(cls, draft_entry):
                 headers=cls.REQUEST_HEADERS,
                 json=payload
             )
-            if response.status_code == 200:
+            if response.status_code == 202:
                 return True
         except Exception:
             logger.error('Assisted tagging send failed, Exception occurred!!', exc_info=True)
             draft_entry.prediction_status = DraftEntry.PredictionStatus.SEND_FAILED
             draft_entry.save(update_fields=('prediction_status',))
         _response = locals().get('response')
+        logger.error(payload)
```

Comment:
It's already in extra, any reason why we have here as well?

Reply:
we are unable to capture extra in logger